source/_components/ebusd.markdown

```diff
@@ -42,7 +45,7 @@ name:
   required: false
   default: ebusd
 monitored_conditions:
-  description: List of condition to monitor.
+  description: List of condition to monitor. Refer to https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/ebusd/const.py for device support table.
```

Comment:
Make it possible to click on the url

Reply:
Fixed. Thanks.

Reply:
I've just talked to @frenck, it's better to omit the link 😉

Reply:
Yes, that was too technical. Fixed by adding explanation to monitored_conditions.